src/loaddb/load_lexer.l

```diff
@@ -501,6 +503,10 @@ using token = cubload::parser::token;
     m_semantic_helper.append_char ('\0');
     // PRINT ("IDENTIFIER %s\n", qstr_Buf_p);
     yylval->string = m_semantic_helper.make_string_by_buffer ();
+	if (er_errid () != NO_ERROR)
```

Comment:
I think handling NULL value for string is more robust code:
if (yylval->string == NULL)
{
    assert (er_errid () != NO_ERROR);
    if (er_errid () == NO_ERROR)
      {
        YY_FATAL_ERROR ("Internal error parsing string");
      }
     else
      {        
          YY_FATAL_ERROR (er_msg ());
      }
}